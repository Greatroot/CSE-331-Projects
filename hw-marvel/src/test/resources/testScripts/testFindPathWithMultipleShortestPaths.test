# Tests that MarvelPaths.findPath() returns the lexicographically first shortest path.

CreateGraph g
AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4
AddNode g n5
AddEdge g n1 n2 e1
AddEdge g n2 n3 e2
AddEdge g n2 n4 e3
AddEdge g n3 n5 e4
AddEdge g n4 n5 e5

FindPath g n1 n5