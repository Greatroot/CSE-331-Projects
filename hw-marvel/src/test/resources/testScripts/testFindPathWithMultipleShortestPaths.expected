# Tests that MarvelPaths.findPath() returns the lexicographically first shortest path.

created graph g
added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g
added node n5 to g
added edge e1 from n1 to n2 in g
added edge e2 from n2 to n3 in g
added edge e3 from n2 to n4 in g
added edge e4 from n3 to n5 in g
added edge e5 from n4 to n5 in g

path from n1 to n5:
n1 to n2 via e1
n2 to n3 via e2
n3 to n5 via e4